js/runner_test.go

```diff
@@ -824,6 +825,13 @@ func TestVUIntegrationHosts(t *testing.T) {
 }
 
 func TestVUIntegrationTLSConfig(t *testing.T) {
+	var unsupportedVersionErrorMsg = "remote error: tls: handshake failure"
+	for _, tag := range build.Default.ReleaseTags {
+		if tag == "go1.12" {
```

Comment:
hmm this will still apply when go 1.13 is released, right?

Reply:
ReleaseTags is list of strings like `go1.x` where  x is from 1 to a number which is the current version. So it have go1.1 , go1.2 .... go1.12 in go 1.12 and will get go1.13 in go 1.13 ;) 
 [source](https://github.com/golang/go/blob/release-branch.go1.12/src/go/build/build.go#L295-L305)